Release notes for V3.0.0-preview2 (#1034)
dotnet/SqlClient: BUILDGUIDE.md

**"Switch.Microsoft.Data.SqlClient.EnableRetryLogic"**

> ## Enabling row version null behaviour

Reviewer: Please use American spelling for Behavior instead of "Behaviour"

> `SqlDataReader` returns a `DBNull` value instead of an empty `byte[]`. To active the legacy behavior, you must enable the following AppContext switch that is cached on first check for the entire process:

Reviewer 2: ```suggestion
`SqlDataReader` returns a `DBNull` value instead of an empty `byte[]`. To enable the legacy behavior, you must enable the following AppContext switch on application startup:
```

> `SqlDataReader` returns a `DBNull` value instead of an empty `byte[]`. To active the legacy behaviour, you must enable the following AppContext switch that is cached on first check for the entire process:

Reviewer: ```suggestion
`SqlDataReader` returns a `DBNull` value instead of an empty `byte[]`. To active the legacy behavior, you must enable the following AppContext switch that is cached on first check for the entire process:
```

> **"Switch.Microsoft.Data.SqlClient.LegacyRowVersionNullBehaviour"**

Reviewer: Is the Switch `LegacyRowVersionNullBehaviour` or `LegacyRowVersionNullBehavior`

Reviewer 3: It's actually `LegacyRowVersionNullBehaviour` https://github.com/dotnet/SqlClient/pull/998
@David-Engel any concerns over UK/US grammar?

Reviewer 4: We should use US grammar.

Reviewer 5: Boooo, but ok i understand.
